resources/bandplan.csv

```diff
@@ -104,11 +104,23 @@
 614000000, 698000000, AM,  10, #902E98BB, Broadcast Television
 698000000, 763000000, AM,  10, #902E98BB, Broadcast Television
 
+# Europe PMR446
+446000000, 446094000, NFM,  6250, #9000FF00, PMR446 Analog 1-8
```

Comment:
The end frequency here should be 446100000 (446.093750 + 6250) according to  https://en.wikipedia.org/wiki/PMR446#Analog_(NFM)_and_digital_TDMA_(DMR)

Reply:
Actually, this is still incorrect. There is 16 PMR446 channels used for analog. and 8 different ones for dPMR446. See for example this: https://cs.wikipedia.org/wiki/PMR_446#Technick%C3%A9_parametry

Reply:
Sorry, also 16 channels for dPMR446, not 8.

Reply:
I see that your linked page mentions 32 dPMR446 channels: "Before January 2018, only the upper 16 channels were allowed for digital FDMA operation."

This may be correct, however I am not sure.

Reply:
I guess it will not be possible to add all those details here, can we try to create some generally useful summary so the MR have chance to get it into `master`?

@AsciiWolf So the spacing above should be 12.5 rather than 6.25, right?

Reply:
https://www.ctu.cz/sites/default/files/obsah/ctu/vseobecne-opravneni-c.vo-r/3/6.2016-9/obrazky/vo-r03-062016-09.pdf
(Page 2)

This is probably more accurate than Wikipedia (and should be valid for the whole European Union).

Reply:
Nice one! Would you be so kind and try to write here some suggestion what would be best to use here by your opinion?

Reply:
Probably the best will be:
```
446006250, 446193750, NFM,  12500, #9000FF00, PMR446 Analog 1-16
446003125, 446196875, NFM,  6250, #9000FF00, PMR446 Digital 1-32
```

Reply:
The 446006250 is center of a channel, should not be here 446000000 to not cut the half of the channel? The same for apply for the upper bound.

Reply:
Good point. :-)